.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,11 @@
-##### What version (commit) do you encounter this issue on?
+#### Do NOT include any confidential information, this issue will be created on a public repo
+#### This section is completed by the issue reporter
+##### On which version(s) do you encounter this issue?
+###### Please provide the output to 'git describe --all' for  all deployments where you have confirmed it occurs.
 
-##### What type of issue is this (bug, feature, documentation, regression)?
+##### What type of issue is this? Please tick the appropriate box.
+- [ ] bug
+- [ ] feature
```

Comment:
I might even be more explicit (unless we're parsing that in other tools or something).

  bug (existing functionality does not work as documented)
  feature (adding functionality)

Or some such thing. Give more clarity, and less chance ot checking the wrong thing.